tls/s2n_server_extensions.c

```diff
@@ -49,7 +49,8 @@
 #define s2n_server_can_send_nst(conn) (s2n_server_sending_nst((conn)) && \
         (conn)->actual_protocol_version < S2N_TLS13)
 
-int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *out)
+/* compute size server extensions send requires */
+uint16_t s2n_server_extensions_send_size(struct s2n_connection *conn)
```

Comment:
Keep this an `int` so we can guard on the total being able to fit in a `uint16_t`

Reply:
ack